packages/console/src/command-history/CommandHistory.test.jsx

```diff
@@ -38,176 +46,192 @@ function mountItems(items) {
-  const wrapper = mountItems(defaultItems);
+  mountItems();
 
-  clickItem(wrapper, 2);
-  clickItem(wrapper, 4, { shiftKey: true });
+  clickItem(2);
+  clickItem(4, { shiftKey: true });
 
-  expect(getItem(wrapper, 1).hasClass('selected')).toBe(false);
-  expect(getItem(wrapper, 2).hasClass('selected')).toBe(true);
-  expect(getItem(wrapper, 3).hasClass('selected')).toBe(true);
-  expect(getItem(wrapper, 4).hasClass('selected')).toBe(true);
-  expect(getItem(wrapper, 5).hasClass('selected')).toBe(false);
+  expectNotSelected(1);
+  expectSelected(2);
+  expectSelected(3);
+  expectSelected(4);
+  expectNotSelected(5);
 
-  clickItem(wrapper, 3);
+  clickItem(3);
 
-  expect(getItem(wrapper, 1).hasClass('selected')).toBe(false);
-  expect(getItem(wrapper, 2).hasClass('selected')).toBe(false);
-  expect(getItem(wrapper, 3).hasClass('selected')).toBe(true);
-  expect(getItem(wrapper, 4).hasClass('selected')).toBe(false);
-  expect(getItem(wrapper, 5).hasClass('selected')).toBe(false);
+  expectNotSelected(1);
+  expectNotSelected(2);
+  expectSelected(3);
+  expectNotSelected(4);
+  expectNotSelected(5);
 });
 
 it('handles selecting multiple items on modifier click', () => {
-  const wrapper = mountItems(defaultItems);
+  mountItems(10);
 
   const selectedItems = [3, 5, 7];
   for (let i = 0; i < selectedItems.length; i += 1) {
-    clickItem(wrapper, selectedItems[i], { ctrlKey: i !== 0 });
+    clickItem(selectedItems[i], { ctrlKey: i !== 0 });
   }
 
   for (let i = 0; i < 10; i += 1) {
     const isSelected = selectedItems.indexOf(i) >= 0;
-    expect(getItem(wrapper, i).hasClass('selected')).toBe(isSelected);
+    if (isSelected) {
+      expectSelected(i);
+    } else {
+      expectNotSelected(i);
+    }
   }
 });
 
 it('handles click and drag', () => {
-  const wrapper = mountItems(defaultItems);
+  mountItems(10);
 
-  dragRange(wrapper, 3, 7);
+  dragRange(3, 7);
 
   for (let i = 0; i < 10; i += 1) {
     const isSelected = i >= 3 && i <= 7;
-    expect(getItem(wrapper, i).hasClass('selected')).toBe(isSelected);
+    if (isSelected) {
+      expectSelected(i);
+    } else {
+      expectNotSelected(i);
+    }
   }
 });
 
 it('handles click and drag with multiples moves on same item', () => {
-  const wrapper = mountItems(defaultItems);
-
-  getItem(wrapper, 3).simulate('mousedown', {});
-  getItem(wrapper, 3).simulate('mousemove', {});
-  getItem(wrapper, 4).simulate('mousemove', {});
-  getItem(wrapper, 4).simulate('mousemove', {});
-  getItem(wrapper, 4).simulate('mousemove', {});
-  getItem(wrapper, 5).simulate('mousemove', {});
-  getItem(wrapper, 6).simulate('mousemove', {});
-  // Drag past the item we want, when we drag back should still be selected
-  getItem(wrapper, 7).simulate('mousemove', {});
-  getItem(wrapper, 6).simulate('mousemove', {});
-  getItem(wrapper, 6).simulate('mouseup', {});
+  mountItems();
+
+  fireEvent.mouseDown(screen.getByText('Command 3'));
```

Comment:
You're entering all these Command text strings manually, it's worth factoring that and getting the command element into their own functions, e.g.
```
function getCommandText(index) {
  return `Command ${index}`;
}

function getCommandItem(index) {
  return screen.getByText(getCommandText(index));
}
```

Reply:
You factored out the string label to `getCommandItem` which is fine; was suggesting you go one step further, and have `getCommandItem` return `screen.getByText(....)` so you don't need to enter screen.getByText everywhere.